gossip-base/src/main/java/org/apache/gossip/manager/GossipManager.java

```diff
@@ -46,14 +48,21 @@
 public abstract class GossipManager {
 
   public static final Logger LOGGER = Logger.getLogger(GossipManager.class);
+  
+  // this mapper is used for ring and user-data persistence only. NOT messages.
+  public static final ObjectMapper metdataObjectMapper = new ObjectMapper() {{
```

Comment:
The goal here was to use only one reference and share between the ring and user-data persistence classes and this was the most logical place to put the reference. I think the problem is that this `ObjectMapper` doesn't really belong in `GossipManager`. 

An approach I would be happy with is to get rid of the static reference here, and have an instance each for the ring and user-data persistence classes. Let me know what you think.